README.md

```diff
@@ -1,20 +1,157 @@
+
+
+`Fairlearn` is an open source unfairness assessment and mitigation toolkit that provides business stakeholders, executives, developers, and data scientists with insights into the unfairness of their model predictions, and techniques to mitigate that. In particular, Fairlearn assists in 
+
+1) Raising awareness of inherent unfairness in model predictions among those involved in developing AI applications by demonstrating harms visited on vulnerable groups
+
+2) Providing state-of-the-art techniques to mitigate the observed unfairness
+
+
+At the assessment phase, Fairlearn provides a dashboard with a rich set of visualizations that uses a set of disparity and accuracy metrics and user-defined protected attributes (e.g., age, gender, ethnicity) to monitor the accuracy and fairness of a model’s predictions.
+
+
+At the mitigation phase, it incorporates state-of-the-art unfairness mitigation techniques to mitigate the unfairness observed at the previous assessment phase. Fairlearn’s mitigation stage supports regression and binary classification models trained on tabular data.
+
+
+
+# <a name="target"></a>
+# Target Audience
+
+
+
+1. Machine Learning Fairness Researchers: ML researchers can easily add their unfairness mitigation techniques to this repository and use the assessment toolkit to load their original and refined model predictions to get insights into how much they have improved the disparity metrics of interest.
+
+
+2. Developers/ ML Engineers/ Data Scientists: Having a variety of curated unfairness mitigations techniques in one place makes it easier for data scientists and (ML) developers to experiment with different unfairness techniques in a seamless manner. The set of rich interactive visualizations allow developers and data scientists to easily visualize the unfairness happening in their machine learning models instead of wasting time and effort on generating customized visualizations.
+
+3. Business Executives: The set of provided visualizations are beneficial for raising awareness among those involved in developing AI applications, allow them to audit model predictions for potential unfairness, and establish a strong governance framework around the use of AI applications.
+
+# <a name="assessment"></a>
+# Fairness Assessment Dashboard 
+The assessment dashboard automatically analyzes a model’s predictions, provides user with a set of insights into how the model is treating different buckets (e.g., female, male, other gender) of a protected attribute/feature (e.g., gender).  
```

Comment:
"provides user" -> either pluralize or make definite
"buckets" -> Shouldn't that be groups?
Again, keep away from protected attribute, unless we're going to reference the specific legal context. I think the sentence could end after "treating different groups (e.g. ...)."